pkg/util/resizefs/resizefs_unsupported.go

```diff
@@ -0,0 +1,25 @@
+// +build !linux
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resizefs
+
+import "fmt"
+
+func ResizeExt(device string, size *resource.Quantity) error {
```

Comment:
Should it implement `NewResizeFs` and with all dummy public functions  so kubelet can be compiled?

Reply:
yeah I need to do that. I will clean it up.